src/custom-types.json

```diff
@@ -743,6 +743,14 @@
       "https://tools.ietf.org/html/rfc7763"
     ]
   },
+  "text/mdx": {
+    "compressible": false,
+    "extensions": ["mdx"],
```

Comment:
Where in https://github.com/mdx-js/specification is the extension `.mdx` stated, or is there another doc that states the extension map?

Reply:
I didn't realize it doesn't specify file extension, I'm sorry. (I'm nailing these mime type PRs, aren't I? 😓) I opened a PR to add the extension to the spec, the team is super responsive now, so it should be a matter of a day or two before it gets reviewed. I'll ping you once it's resolved.

Reply:
It's no problem. That's mainly why the recommended way to add new types and stuff is https://github.com/jshttp/mime-db#adding-custom-media-types and not through PRs; there is a form there that asks for all the required information.